script.js

```diff
@@ -1,4 +1,117 @@
+// Define current player
+var currentPlayer = 1;
+
+// Function for output
+var myOutputValue = "";
+
+// Define variables to keep track of player's number
+var playerNumber = [];
+var playerOneNum = playerNumber[0];
+var playerTwoNum = playerNumber[1];
+
+// Function to track game mode
+var modeRollDice = "Roll dice mode";
+var modeChoseOrder = "Chose order of numbers";
+var modeCurrent = modeRollDice;
+
+// Function to keep track of dice rolled
+var diceNumberRolled = [];
+var diceRollA = "";
+var diceRollB = "";
+
+// Combination of numbers
+// var combineNumbers = "";
+var combineNumbersAB = Number(diceRollA) + Number(diceRollB);
+var combineNumbersBA = Number(diceRollB) + Number(diceRollA);
+
+// Function to get random dice number
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // Select game mode
+  var gameCounter = 0;
+  if (modeCurrent == modeRollDice) {
+    myOutputValue = rollDiceTwice(gameCounter);
+    modeCurrent = modeChoseOrder;
+  }
+  // Choose order of numbers
+  else if (modeCurrent == modeChoseOrder) {
+    // If Player chooses AB arrangement
+    var combinedNumbers = String(diceRollA) + String(diceRollB);
+    var myOutputValue = `The number you've chose is ${combinedNumbers} <br><br> Click 'submit' to continue. `;
```

Comment:
I don't suggest creating variables within conditional blocks like `if`s, `else`s or `else if`s, unless you only intend to use them in said blocks. View my comment above for what I suggest instead.